Dockerfile

```diff
@@ -32,9 +34,7 @@ ARG DEBUG=false
 
 RUN if [ "${DEBUG}" = "true" ]; then pip install -r requirements-dev.txt; pip install -e .[all]; else pip install .[all]; fi;
 
-RUN adduser --uid 1000 --disabled-password --gecos '' reanauser && \
-    chown -R reanauser:reanauser /code
-USER reanauser
 EXPOSE 5000
 ENV FLASK_APP=/code/reana_server/app.py
-CMD ["flask", "run", "--host=0.0.0.0"]
+CMD flask users create info@reana.io &&\
```

Comment:
`users create` needed?

Reply:
It is emulating what `workflow-controller` used to do. If we don't do it here we have to add an additional step to the installation instructions or to `reana-cluster init` cc @dinosk

Reply:
OK, let's keep it... We'll change things WRT user registration and login later anyway...